mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java

```diff
@@ -252,7 +252,7 @@ public void onClickSubmitButton() {
             clientPayload.setExternalId(etClientExternalId.getEditableText().toString());
         }
 
-        if (!clientStaff.isEmpty()) {
+        if (!staffList.isEmpty()) {
```

Comment:
Error logcat for the issue can be found [here](https://gist.github.com/rdeveloperIITR/b5f886ae1599c2ea425934b6d4d7bfeb)